src/9.negotiation.js

```diff
@@ -607,8 +607,13 @@ export function /*9.2.8 */SupportedLocales (availableLocales, requestedLocales,
             writable: false, configurable: false, value: subset[P]
         });
     }
-    // "Freeze" the array so no new elements can be added
-    defineProperty(subset, 'length', { writable: false });
+    
+    // Wrap in try catch for older browsers that don't support setting length of
```

Comment:
try running the unit tests with this operation removed. If everything passes, we can remove it. Otherwise I will be fine with the try/catch.